tools/type_whisperer/types.proto

```diff
@@ -2,20 +2,42 @@ syntax = "proto3";
 
 package tools.type_whisperer;
 
-// Description for a single type.
-message TypeDescription {
+// Field level type information.
+message FieldDescription {
```

Comment:
I feel we're adding more and more to the type_db, but some of the information is really just duplicating proto descriptor. Since we will have a few more annotation, WDYT about just use protobuf DescriptorPool from a DescriptorSet generated from full source tree (which is easily doable from proto_library or other mean) for those information and minimize typedb to the DAG of type upgrading?

Reply:
Yeah, I think this could work. I can take a look at this when I have all tests passing for v3alpha.

Reply:
Done, PTAL.